src/core/views.py

```diff
@@ -224,20 +224,30 @@ def get_reset_token(request):
     :return: HttpResponse object
     """
     new_reset_token = None
+    form = forms.GetResetToken()
 
     if request.POST:
-        email_address = request.POST.get('email_address')
-        messages.add_message(request, messages.INFO, 'If your account was found, an email has been sent to you.')
-        try:
-            account = models.Account.objects.get(email__iexact=email_address)
-            logic.start_reset_process(request, account)
-            return redirect(reverse('core_login'))
-        except models.Account.DoesNotExist:
-            return redirect(reverse('core_login'))
+        form = forms.GetResetToken(
+            request.POST,
+        )
+        if form.is_valid():
+            email_address = form.cleaned_data.get("email_address")
+            messages.add_message(
+                request, 
+                messages.INFO,
+                'If your account was found, an email has been sent to you.',
```

Comment:
We should mark this text for translation as we've done with the email label in forms.py.

Reply:
I've updated this, but am struggling to change the language in order to test it.  I can confirm the change I've made doesn't seem to affect the English version, but I can't work out whether it is functional in terms of translation.

Reply:
> I've updated this, but am struggling to change the language in order to test it. I can confirm the change I've made doesn't seem to affect the English version, but I can't work out whether it is functional in terms of translation.

In this case that is enough, it wont have a translation so will always show the English version until that is loaded in.